### PR TITLE
Scale cli pre rollout tasks

### PR DESCRIPTION
**Pull request by rtprio**
This ensures that the cli deployment config is scaled before trying to run post rollout tasks.

### CODE REVIEW

File: images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh [hunk 1]
```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+# if there is a deploymentconfig for the given service
+if [[ "oc -n ${OPENSHIFT_PROJECT} get deploymentconfigs -l service=${SERVICE_NAME}" ]]; then
```
Comment (Schnitzel) [inline images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh]: @rtprio 
this looks like a check for a string for me which will always be true?

File: images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+  DEPLOYMENTCONFIG=$(oc -n ${OPENSHIFT_PROJECT} get deploymentconfigs -l service=${SERVICE_NAME} -o name)
+  # If the deploymentconfig is scaled to 0, scale to 1
+  if [[ $(oc -n ${OPENSHIFT_PROJECT} get ${DEPLOYMENTCONFIG} -o go-template --template='{{.status.replicas}}') == "0" ]]; then
+
+    oc -n ${OPENSHIFT_PROJECT} scale --replicas=1 ${DEPLOYMENTCONFIG} >/dev/null 2>&1
+    # wait until the scaling is done
+    while [[ ! $(oc -n ${OPENSHIFT_PROJECT}-n ${OPENSHIFT_PROJECT} get ${DEPLOYMENTCONFIG} -o go-template --template='{{.status.readyReplicas}}') == "1" ]]
```
Comment (Schnitzel) [inline images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh]: `-n ${OPENSHIFT_PROJECT}-n ${OPENSHIFT_PROJECT}`

**Review by Schnitzel — CHANGES_REQUESTED** [review]

File: images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh (hunk 1 above)
Comment (rtprio) [reply images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh]: corrected.

File: images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh (hunk 2 above)
Comment (rtprio) [reply images/oc-build-deploy-dind/scripts/exec-post-rollout-tasks-run.sh]: mispaste

**Comment by rtprio** [comment]
Does this look better @Schnitzel ?

**Review by Schnitzel — APPROVED** [review]

**Pull request merged**